### DISCUSSION TITLE
Plantronics Hub icon

### DISCUSSION THREAD

**Discussion by 33blocks** [elrumo/macOS_Big_Sur_icons_replacements#1302] [Icon request]
Can you add a "Plantronics Hub" icon

**Comment by OMGzilla-git** [comment]
Someone posted a couple versions based on the iOS app's icon, which look great! I went ahead and made one based on the Mac app's icon as an alternative. Theirs are available, mine should be in the next day or two. You can grab it off my profile if desired for now from the Waiting section.
https://macosicons.com/u/OMGzilla
